examples-generated/rds/dbinstance.yaml

```
@@ -0,0 +1,23 @@
+# This example manifest is auto-generated, and has not been tested.
+# Please make the necessary adjustments before using it.
+apiVersion: rds.aws.jet.crossplane.io/v1alpha1
+kind: DBInstance
```

```suggestion
kind: Instance
```

Are we able to use the final configurations with all custom overrides before calculating these?

---

Done. File has been renamed as `examples-generated/rds/instance.yaml` upon regeneration.